swagger/Platfrom API/bizaiChats.json

```diff
@@ -0,0 +1,407 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "Platform Management",
+    "description": "",
+    "contact": {},
+    "version": "1.0.0"
+  },
+  "servers": [
+    {
+      "url": "https://api.vcita.biz/v3/"
+    }
+  ],
+  "tags": [
+    {
+      "name": "chatmng",
+      "description": ""
+    }
+  ],
+  "paths": {
+    "/v3/bizai-chats": {
+      "get": {
+        "operationId": "ChatsController_getChats",
+        "summary": "Get all chats",
+        "description": "Get a list of all ongoing chat sessions",
+        "parameters": [
+          {
+            "in": "query",
+            "name": "page",
+            "schema": {
+              "type": "integer",
+              "default": 1
+            },
+            "description": "Page number of the results."
+          },
+          {
+            "in": "query",
+            "name": "per_page",
+            "schema": {
+              "type": "integer",
+              "default": 25,
+              "maximum": 100
+            },
+            "description": "How many items to return per page."
+          },
+          {
+            "in": "query",
+            "name": "sort",
+            "schema": {
+              "type": "string",
+              "default": "updated_at:desc"
+            },
+            "description": "Sort by fields, e.g., \"created_at:asc,updated_at:desc\"."
+          },
+          {
+            "in": "query",
```

Comment:
You need to be explicit about supported filters and set them up as parameters. see documentation about filter (its different than the current standard.

Reply:
Read this section in the documentation:
https://myvcita.atlassian.net/wiki/spaces/IT/pages/2941976643/API+v3+Standards+-+WIP#Filtering
I am not sure which example to you mean

Reply:
The `subscription.json` file